src/index.ts

```diff
@@ -80,12 +80,13 @@ async function lookup(store: Store) {
  */
 function isOutOfStock(domText: string, oosLabels: string[]) {
 	const domTextLowerCase = domText.toLowerCase();
-	let result = false;
 	for (const oosLabel of oosLabels) {
-		result = domTextLowerCase.includes(oosLabel.toLowerCase());
+		if (domTextLowerCase.includes(oosLabel.toLowerCase())) {
+			return true;
+		}
 	}
 
-	return result;
+	return false;
```

Comment:
If you wanna take it one step further, let's just do this:

```typescript
const domTextLowerCase = domText.toLowerCase();
return oosLabels.some(label => domTextLowerCase.includes(label));
```

Looks great otherwise!

Reply:
Yup, I thought it was merging in the results. Probably could still do that (not sure what the return of .includes is), but just breaking the function once you find our first in-stock hit makes more sense anyway!

Reply:
Nice to see more languages are adopting more functional styling; makes things like this more elegant.